habitat/tasks/nav/nav_task.py

```diff
@@ -491,6 +510,29 @@ def reset_metric(self, episode):
             self._map_resolution,
         )
         self._previous_xy_location = (a_y, a_x)
+        self._draw_shortest_path()
+
+    def _draw_shortest_path(self):
+        for prev_pt, next_pt in zip(
+            self._shortest_path_points[:-1], self._shortest_path_points[1:]
+        ):
+            if (
+                self._top_down_map[prev_pt[0], prev_pt[1]]
+                != maps.MAP_SOURCE_POINT_INDICATOR
+            ):
+                color = 7
```

Comment:
a comment here too, what does 7 refer to? Maybe give a reference if someone wants to change color.